galasa-simbank-tests/dev.galasa.simbank.tests/src/main/java/dev/galasa/simbank/tests/SimBankIVT.java

```diff
@@ -59,9 +57,13 @@ public void testNotNull() {
     }
 
     /**
-     * Test which checks the initial balance of an account, uses the webservice to
-     * credit the account, then checks the balance again. The test passes if the
-     * final balance is equal to the old balance + the credited amount.
+     * Test which checks the Simbank application is working and can be logged into using the 3270 terminal.
```

Comment:
"Simbank application is working" - careful, that sounds like your testing the whole application. Maybe say the systems in which the Simbank application is installed in is reachable